stubs/openpyxl/openpyxl/worksheet/worksheet.pyi

```diff
@@ -24,106 +33,106 @@ class Worksheet(_WorkbookChild):
+    def freeze_panes(self) -> str | None: ...
     @freeze_panes.setter
     def freeze_panes(self, topLeftCell: Incomplete | None = ...) -> None: ...
-    def cell(self, row, column, value: Incomplete | None = ...): ...
-    def __getitem__(self, key): ...
-    def __setitem__(self, key, value) -> None: ...
-    def __iter__(self): ...
-    def __delitem__(self, key) -> None: ...
+    def cell(self, row: int, column: int, value: str | None = ...) -> Cell: ...
+    def __getitem__(self, key: str | int | slice) -> Cell | tuple[Cell, ...]: ...
+    def __setitem__(self, key: str, value: str) -> None: ...
+    def __iter__(self) -> Iterator[Cell]: ...
+    def __delitem__(self, key: str) -> None: ...
     @property
-    def min_row(self): ...
+    def min_row(self) -> int: ...
     @property
-    def max_row(self): ...
+    def max_row(self) -> int: ...
     @property
-    def min_column(self): ...
+    def min_column(self) -> int: ...
     @property
-    def max_column(self): ...
-    def calculate_dimension(self): ...
+    def max_column(self) -> int: ...
+    def calculate_dimension(self) -> str: ...
     @property
-    def dimensions(self): ...
+    def dimensions(self) -> str: ...
     def iter_rows(
         self,
-        min_row: Incomplete | None = ...,
-        max_row: Incomplete | None = ...,
-        min_col: Incomplete | None = ...,
-        max_col: Incomplete | None = ...,
+        min_row: int | None = ...,
+        max_row: int | None = ...,
+        min_col: int | None = ...,
+        max_col: int | None = ...,
         values_only: bool = ...,
-    ): ...
+    ) -> Generator[Cell | str | float | datetime | None, None, None]: ...
     @property
-    def rows(self): ...
+    def rows(self) -> Generator[Cell, None, None]: ...
     @property
-    def values(self) -> Generator[Incomplete, None, None]: ...
+    def values(self) -> Generator[str | float | datetime | None, None, None]: ...
     def iter_cols(
         self,
-        min_col: Incomplete | None = ...,
-        max_col: Incomplete | None = ...,
-        min_row: Incomplete | None = ...,
-        max_row: Incomplete | None = ...,
+        min_col: int | None = ...,
+        max_col: int | None = ...,
+        min_row: int | None = ...,
+        max_row: int | None = ...,
         values_only: bool = ...,
-    ): ...
+    ) -> Generator[Cell | str | float | datetime | None, None, None]: ...
     @property
-    def columns(self): ...
-    def set_printer_settings(self, paper_size, orientation) -> None: ...
-    def add_data_validation(self, data_validation) -> None: ...
-    def add_chart(self, chart, anchor: Incomplete | None = ...) -> None: ...
-    def add_image(self, img, anchor: Incomplete | None = ...) -> None: ...
-    def add_table(self, table) -> None: ...
+    def columns(self) -> Generator[Cell, None, None]: ...
+    def set_printer_settings(self, paper_size: int, orientation: None | Literal["default", "portrait", "landscape"]) -> None: ...
```

Comment:
I think `paper_size` can also be `None`, if I understand the `openpyxl` descriptor mechanisms correctly: https://foss.heptapod.net/openpyxl/openpyxl/-/blob/branch/3.0/openpyxl/worksheet/page.py#L19

```suggestion
    def set_printer_settings(self, paper_size: int | None, orientation: None | Literal["default", "portrait", "landscape"]) -> None: ...
```